files/zh-cn/glossary/value/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: 值
+slug: Glossary/Value
+---
+
+在数据或对象{{Glossary("Wrapper", "包装器")}}的概念下，值是对象包装器包含的{{Glossary("Primitive", "原始值")}}。在{{Glossary("Variable", "变量")}}或{{Glossary("Property", "属性")}}的概念下，值可以是原始类型或{{Glossary("Object reference", "对象引用")}}。
```

Comment:
markdown bold syntax seems not to work here

Reply:
Please still wrapper this with `bold`. The problem is caused by smartlink helper function in mdn/yari (when the link need to fallback to en-US).

Reply:
Okay.

Reply:
```suggestion
在数据或对象**{{Glossary("Wrapper", "包装器")}}**的概念下，值是对象包装器包含的**{{Glossary("Primitive", "原始值")}}**。在**{{Glossary("Variable", "变量")}}**或**{{Glossary("Property", "属性")}}**的概念下，值可以是原始类型或**{{Glossary("Object reference", "对象引用")}}**。
```